yarn-project/stdlib/src/interfaces/aztec-node.ts

```diff
@@ -552,7 +552,7 @@ export function createAztecNodeClient(
   url: string,
   versions: Partial<ComponentsVersions> = {},
   fetch = makeFetch([1, 2, 3], false),
-  batchWindowMS = 25,
+  batchWindowMS = 0,
```

Comment:
A small change @nventuro suggested in order to reduce lag on PXE operations until it is refactored to do batching. I've bundled this change into the same PR in order to reduce merge-queue duration